integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -42,11 +42,11 @@
                     TimestampGen(start=datetime(1677, 9, 22, tzinfo=timezone.utc), end=datetime(2262, 4, 11, tzinfo=timezone.utc))]
 parquet_basic_struct_gen = StructGen([['child'+str(ind), sub_gen] for ind, sub_gen in enumerate(parquet_basic_gen)])
 
-parquet_write_gens_list = [
-    parquet_basic_gen,
-    pytest.param(parquet_basic_gen + [decimal_gen_default,
-                  decimal_gen_scale_precision, decimal_gen_same_scale_precision, decimal_gen_64bit],
-                 marks=pytest.mark.allow_non_gpu("CoalesceExec"))]
+parquet_write_gens_list = [parquet_basic_gen,
+                           pytest.param([decimal_gen_default, parquet_basic_struct_gen,
+                           StructGen([['child0', StructGen([['child1', byte_gen]])]]),
+                           decimal_gen_scale_precision, decimal_gen_same_scale_precision, decimal_gen_64bit],
+                                        marks=pytest.mark.allow_non_gpu("CoalesceExec"))]
```

Comment:
Why is `CoalesceExec` not on the GPU? and is there an open issue that will put it on the GPU? If not we should file one and reference it here. If so we should just reference it here.

Reply:
I assume this is fixed by #2530 which added struct support for coalesce.

Reply:
I have updated the PR to not allow Coalesce with Structs to run on CPU

Reply:
no, its not refflecting it here for some reason, unless you go to the last commit

Reply:
Are you referring to a commit that has yet to be pushed to his PR? Because I still see it in https://github.com/NVIDIA/spark-rapids/pull/2514/files and at https://github.com/razajafri/spark-rapids-1/blob/parquet-doc/integration_tests/src/main/python/parquet_write_test.py#L49

Reply:
OK you are seeing the latest, I wasn't seeing it until I looked at the commit specifically.

So that is needed because CoalesceExec doesn't support Decimals yet

Reply:
> So that is needed because CoalesceExec doesn't support Decimals yet

#2531 explicitly states it adds decimal support and appears to be testing it.  So if that is somehow not working in practice an issue needs to be filed and fixed.

Reply:
Sorry for wasting your time. I didn't see `_commonTypes` was added and thought it was `TypeSig.commonCudfTypes`